src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,329 @@
+in Munich's MarienPlatz. 
+
+## How are media Files put together?
+
+Before I start manipulating media files, I want to talk a bit about how media
+files are put together. I think of them as being like an onion. The file that
+you see in your operating system shell is a _container_, identified by a file
+extension (mp4, webm, etc.). The container houses one or more _streams_ of up to
+four types, specifically audio, video, captions, and data. Although containers
+will typically contain only one stream of each type, some container types, webm
+for example, allow multiple streams of the same type. Within the audio and video
+streams, the actual data is compressed using a _codec_. As we'll see later, the
+distinction between a container and a codec is import as files with the same
+container can have their contents encoded with different codecs.
+
+(If these terms are new
+to you, I explain them in
+[Media file characteristics](/web/fundamentals/media/manipulating/applications#media_file_characteristics).)
+
+The image below illustrates this. On the left is the basic structure. On the
+right are the specifics of that structure for an mp4. I'll explain file
+manipulation by moving downward through these layers.
+
+![Media container onion](images/media-container-onion.png)
+
+## Change the container
+
+Let's start by changing the file container. You'll recall that we're starting
+with a file that has an mov extension. I'm going to use ffmpeg to change the
+container type from mov to mp4 and webm. In actual practice, you would likely
+specify a codec at the same time. For this lesson, I'm letting ffmpeg use it's
+defaults.
+
+    ffmpeg -i glocken.mov glocken.mp4
+
+Creating a webm file is a bit more complicated. ffmpeg has no trouble converting
+a mov file to a webm file that will play in Chrome and Firefox. For whatever
+reason, the file created using ffmpeg's defaults doesn't quite conform to the
+webm spec. (For the curious, it sets a `DisplayUnit` size that isn't defined by
+the webm spec.) Fortunately, I can fix this using a video filter. Do so with the
+`-vf` flag and the setsar filter.
+
+    ffmpeg -i glocken.mov -vf setsar=1:1 glocken.webm
+
+Webm takes quite a bit longer to create than mp4. This isn't surprising when
+you look at the results. While mp4 compresses to about a quarter of the original
+file's size, webm compresses to 1% of the original file's size.
+
+    -rw-r--r--  1 jmedley  eng  12080306 May  9 14:34 glocken.mov
+    -rw-r--r--  1 jmedley  eng  10407497 May  9 14:52 glocken.mp4
+    -rw-r--r--  1 jmedley  eng    392263 May  9 14:49 glocken.webm
+
+## Split the streams
+
+When you package media resources, for HLS and DASH, you'll need to separate the
+video and audio streams. Splitting the audio and video streams of a file is
+often referred to as **demultiplexing** or **demuxing** for short. Professional
+content creators record audio and video to separate files to begin with. If
```

Comment:
creators **_often_** record audio and video to separate files

Plenty of high-end devices create single files with both audio and video